.eslintrc

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": ["standard"]
```

Comment:
Add eslint as a devDependency otherwise all the editors will fallback into the `global` eslint package and maybe the user either do not have it or maybe has an eslint version which is in conflict with the module version